lib/common/results.c

```diff
@@ -318,6 +318,10 @@ static const struct pcmk__rc_info {
       "Network code configuration error",
       -pcmk_err_generic,
     },
+    { "pcmk_rc_bz2_software",
+      "Decompression code configuration error",
```

Comment:
maybe "Compression/decompression error" (saying configuration suggests the user can do something about it)